docs/operator_guide.md

```diff
@@ -28,7 +28,7 @@ The Basics
 
 In order for FIAAS to function in a cluster, some basics needs to be in place
 
-* Autoscaling based on CPU metrics requires Heapster, deployed in the kube-system namespace
+* Autoscaling based on CPU metrics requires the metrics API (provided by Heapster or metrics-server), deployed in the kube-system namespace
```

Comment:
I'm not sure heapster or metrics-server *have* to be deployed in the kube-system namespace. Maybe remove that part?